app/stac_api/admin.py

```diff
@@ -53,7 +63,8 @@ class ItemAdmin(admin.ModelAdmin):
                     'properties_datetime',
                     'properties_start_datetime',
                     'properties_end_datetime',
-                    'properties_title'
+                    'properties_title',
+                    'properties_eo_gsd'
```

Comment:
This is a readonly field and is not allowed to be here

Reply:
isn't it the task to show them?

Reply:
If we can display these read-only fields in the admin it would be nice. If it's not easily possible and needs ugly hacks we leave it.

Reply:
`properties_eo_gsd` will disappear from the Item class, as soon as https://github.com/geoadmin/service-stac/pull/107 is merged ;-)